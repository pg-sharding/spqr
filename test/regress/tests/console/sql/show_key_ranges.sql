CREATE DISTRIBUTION ds1 COLUMN TYPES integer;
CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1 FOR DISTRIBUTION ds1;
CREATE KEY RANGE krid2 FROM 11 ROUTE TO sh1 FOR DISTRIBUTION ds1;

SHOW key_ranges;

DROP DISTRIBUTION ALL CASCADE;
DROP KEY RANGE ALL;