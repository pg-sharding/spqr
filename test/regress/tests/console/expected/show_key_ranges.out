
		SQPR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

CREATE KEY RANGE krid1 FROM 1 ROUTE TO sh1;
         add key range          
--------------------------------
 created key range with bound 1
(1 row)

CREATE KEY RANGE krid2 FROM 11 ROUTE TO sh1;
          add key range          
---------------------------------
 created key range with bound 11
(1 row)

SHOW key_ranges;
 Key range ID | Shard ID | Dataspace ID | Lower bound 
--------------+----------+--------------+-------------
 krid1        | sh1      | default      | 1
 krid2        | sh1      | default      | 11
(2 rows)

DROP DATASPACE ALL CASCADE;
 drop dataspace 
----------------
(0 rows)

DROP SHARDING RULE ALL;
   drop sharding rule   
------------------------
 dropped sharding rule 
(1 row)

DROP KEY RANGE ALL;
 drop key range 
----------------
(0 rows)

